bliss/data_augmentation.py

```diff
@@ -0,0 +1,117 @@
+import random
+
+import numpy as np
+import torchvision as TF
+
+from bliss.catalog import TileCatalog
+
+
+def augment_data(tile_catalog, image, deconv_image=None):
+    origin_tile = TileCatalog(4, tile_catalog)
+    origin_full = origin_tile.to_full_params()
+
+    num_transform_list = [1, 2, 3]
+    num_transform = random.choices(num_transform_list, weights=[0.7, 0.2, 0.1], k=1)[0]
+    transform_list = ["vflip", "hflip", "rotate90", "rotate180", "rotate270", "shift"]
+
+    aug_method = np.random.choice(
+        transform_list, num_transform, p=(0.1, 0.1, 0.1, 0.1, 0.1, 0.5), replace=False
+    )
+
+    aug_image, aug_tile, aug_deconv = origin_full, image, deconv_image
+    for i in aug_method:
```

Comment:
`i` usually denotes an integer. Would you use a more descriptive variable name here?